### PR TITLE
React: On login and signup page the logo present at top right should lead to home page when clicked

### PR DESCRIPTION
**Pull request by niloferh**
Addresses issue #203 

Changes (in Login.jsx and SignUp.jsx):
- Created a click handler function that uses `useNavigate` to redirect to the home page
- Added an `onClick` event to the logo so that the click handler is invoked
- Set the cursor to pointer when hovering over logo to let users know it is clickable 
- Set `zIndex: 1` to bring logo element to the front (otherwise only part of the logo area was clickable)

See videos below for the implemented functionality for login and signup pages:

https://github.com/user-attachments/assets/806bf1aa-7577-4b5a-9a8a-3bc0c42328e4

https://github.com/user-attachments/assets/bb3203b9-7433-40a7-91f6-0d6ef67570d1

### CODE REVIEW

**Comment by AmanNegi** [comment]
@niloferh, LGTM! Are you familiar with TailwindCSS, if possible could you directly apply classes instead of using the style attribute?


> Ps: chatgpt might help you achieve that easily.

File: frontend/react/src/pages/Auth/presentation/Login.jsx [hunk 1]
```diff
@@ -54,11 +54,20 @@ function Login() {
...
   return (
     <>
       <section className="float-right relative h-screen w-screen lg:w-[40%]">
-        {/* Top Left Icon and Text */}
-        <div className="absolute right-3 top-3">
+        {/* Top Right Icon and Text */}
+        <div 
+          className="absolute right-3 top-3" 
+          onClick={handleLogoClick} 
+          style={{ cursor: "pointer", zIndex: 1 }}
```
Comment (AmanNegi) [inline frontend/react/src/pages/Auth/presentation/Login.jsx:69]: Could you directly incorporate this into the `className` attribute, using TailwindCSS(the project is already using TailwindCSS).

File: frontend/react/src/pages/Auth/presentation/SignUp.jsx [hunk 2]
```diff
@@ -73,11 +73,20 @@ function SignUp() {
...
   return (
     <>
       <section className="float-right relative h-screen w-screen lg:w-[40%]">
-        {/* Top Left Icon and Text */}
-        <div className="absolute right-3 top-3">
+        {/* Top Right Icon and Text */}
+        <div 
+          className="absolute right-3 top-3" 
+          onClick={handleLogoClick} 
+          style={{ cursor: "pointer", zIndex: 1 }}
```
Comment (AmanNegi) [inline frontend/react/src/pages/Auth/presentation/SignUp.jsx:88]: Similarly here.

File: frontend/react/src/pages/Auth/presentation/SignUp.jsx [hunk 3]
```diff
@@ -73,11 +73,20 @@ function SignUp() {
...
+
   return (
     <>
       <section className="float-right relative h-screen w-screen lg:w-[40%]">
-        {/* Top Left Icon and Text */}
-        <div className="absolute right-3 top-3">
+        {/* Top Right Icon and Text */}
+        <div 
+          className="absolute right-3 top-3" 
+          onClick={handleLogoClick}
```
Comment (AmanNegi) [inline frontend/react/src/pages/Auth/presentation/SignUp.jsx:87]: No need to create a seperate function just for a navigate, feel free to use anonymous function in both login and signup pages.

**Review by AmanNegi — CHANGES_REQUESTED** [review]
Hey there could you make the requested changes, feel free to ask if you have any queries.

**Comment by niloferh** [comment]
Hi @AmanNegi, I've made the changes you requested

**Comment by AmanNegi** [comment]
LGTM @niloferh! Thanks a lot for your contribution. ✨

> You have been kind patient, and included relevant details in your PR and commits, a true spirit for open souce. Thanks for your valuable time! ✨🤗

**Review by AmanNegi — APPROVED** [review]

**Pull request merged**